### ISSUE TITLE
Star  matching does not work '*'

### ISSUE THREAD

**Issue by nicko-winner** [valendres/playwright-msw#81]
`*` pattern matches do not work as documented in MSW, when using playwright-msw.

Used to work in version: 0.2.2

After upgrades to: 
1. playwright-msw 2.2.1 msw 1.x and 
1. playwright-msw 3.0.0 msw 2.0.1 

```js
http.get('*', ({ request }) => { ... }
```
Results in: "SyntaxError: Invalid regular expression: /^(\w+://[^/]+)?*\/?(\?.*)?$/: Nothing to repeat"

```js
http.get('/some/url*', ({ request }) => { ... }
```
Will not match URL

```js
http.get('/so*/url*', ({ request }) => { ... }
```
Will not match URL

URLs without star matching work great.

**Comment by nicko-winner** [comment]
https://github.com/valendres/playwright-msw/pull/84

**Comment by valendres** [comment]
Thanks for raising this issue and creating a PR @nicko-winner. Not being able to do wildcard matching is definitely unintentional. Looks like there's a minor type issue on the PR, but that's okay. I'll help get this merged and released ASAP.

**Comment by valendres** [comment]
This issue has been released in `playwright-msw@3.0.1`

**Comment by nicko-winner** [comment]
Thanks for the help getting that merged. Appreciate all your work on this project, and the recent support for MSW 2.0.
